web-ui/src/main/resources/catalog/components/filestore/partials/filestore.html

```diff
@@ -16,7 +16,7 @@
           {{r.id.split('/').splice(2).join('/') | decodeURIComponent}}
         </a>
       </td>
-      <td data-ng-if="r.version.length > 0">
+      <td style="min-width:50px" data-ng-if="r.version.length > 0">
```

Comment:
It is best to use classes instead of styles.
I would recommend using 
`class="text-nowrap"`

Reply:
Tested by injected from Webtool. It works as well. I will commit it soon.

![image](https://user-images.githubusercontent.com/74916635/107663451-64904e00-6c59-11eb-8dae-3fa119b6bba2.png)